### ISSUE TITLE
Clarification of Group Income & okTurtles purpose/mission/goals

### ISSUE THREAD

**Issue by sdegutis** [okTurtles/group-income#1001]
When I first joined the Group Income project, I was under the impression that it was just a web app, whose purpose is to help groups of friends to share a portion of their abundance with their friends who are in need. The home page and the FAQ and the video shorts gave me that understanding.

After several discussions with @taoeffect, I came to know that there are many other underlying philosophical goals of the project. Many of these have long-reaching effects, such as the deeply invasive presence of SBP widespread in the project, and the presence of real-time chat functionality in the app.

Through these discussions, and reading through the blog post [Basic Security: Our Real Goal](https://groupincome.org/2017/03/basic-security-our-real-goal/), I think I have a better understanding of the philosophies behind okTurtles and all its underlying projects, including Group Currency and Group Income.

I think it can best be described by the word "security" in all its various meanings:

1. Security as in having basic needs met, such as food and water. The blog post uses the examples of a 10 year old or disabled person having access to shelter and food.

2. Security as in having protection from outside threats. This is the reason for the goals of decentralization and end-to-end encryption, which protect against outside spying, etc.

3. Security as in having extra money for non-needs. The blog post describes someone wanting to spend a year learning math and physics, which the "damange that money as a tool creates" prevents them from freely doing.

With these in mind, okTurtles wants to build software that is *secure* from hacking, that helps to provide *basic security* by enabling the sharing of money for basic needs, and wants to provide *extra security*, presumably by helping to create a society in which money is no obstacle to anything.

Group Income seems to have three roles in this overall plan:

1. To create a money splitting web app, which [this thumbnail](https://groupincome.org/wp-content/uploads/2017/01/what-is-groupincome-thumbnail-300x169@2x.jpg) perfectly encapsulates and describes (excellent job, whoever made it). This seems to have been the original goal.

2. At some point along the lines, creating a protocol became more important than creating the app. The idea being, when you have an open protocol such as XMPP or SMTP, it can outlive specific implementations, and can evolve by real-life usage.

3. To create a secure software platform that can be its own entire island, incorporating every single thing anyone would need a computer to do, but in a completely secure (from outside threats) environment, using decentralization, end-to-end encryption, and SBP for this purpose. (SBP was intended to create a secure OS, and that seems to be one of the biggest motivations for incorporating it so deeply into Group Income's source code.)

**Comment by sdegutis** [comment]
A lot of these philosophies seem to come from the idea that, in order to ensure that everyone's basic needs are met, _the people_ need to take control of the _creation_ and _distribution_ of money. However, these responsibilities relating to money are traditionally (and logically) the right and role of _governments_, which are simply organizations made *of* the people, and exist *for* the common good of the people.

**Comment by taoeffect** [comment]
> However, these responsibilities relating to money are traditionally (and logically) the right and role of _governments_, which are simply organizations made *of* the people, and exist *for* the common good of the people.

Yes, and due to population growth we are [hitting scaling boundaries, resulting in problems](https://fixingtao.com/2016/04/lets-end-hotdog-worship-in-america/).

**Comment by sdegutis** [comment]
But what I'm saying is, you're trying to fix *fundamental problems* in society caused by governments *not working properly*, but you're not doing so by creating a new government, but using mostly technology.

**Comment by sdegutis** [comment]
Never mind, this is not the right issue to discuss the merit of any of these goals.

**Comment by taoeffect** [comment]
I think you did a great job in this issue summarizing our mission and goals. I'm not sure if our Github issues is the best place to post this though, and am open to hearing concrete ideas about where this should appear, and how it should appear.

Our issues need to be formatted in Problem/Solution style. Is this something you can do? Otherwise, is it OK if I close this issue for now and we can return to it later?

**Comment by sdegutis** [comment]
Sure, here’s the problem/solution way of rephrasing what I said above.

Problem: the real goals and philosophies of group income and okTurtles are not prominent on the website, and only part of them are present in the blog post, whereas part was discovered through multiple conversations and external links.

Solution: put it prominently on the website, at least in the FAQ if not the main page
